epicsdbbuilder/dbd.py

```diff
@@ -108,13 +109,16 @@ def LoadDbdFile(dbdfile, on_use = None):
         os.chdir(dirname)
 
     # We add <epics_base>/dbd to the path so that dbd includes can be resolved.
+    separator = ':'
+    if platform.system() == 'Windows':
+        separator = ';'
+
     status = mydbstatic.dbReadDatabase(
-        ctypes.byref(_db), filename, '.:%s/dbd' % _epics_base, None)
+        ctypes.byref(_db), filename, separator.join(['.', os.path.join(_epics_base, 'dbd')]), None)
```

Comment:
Please split this line to fit within 80 columns.  I suggest splitting after `filename,` and aligning `separator` with `ctypes` on the line above.